apps/marketplace/components/validators.js

```diff
@@ -35,15 +35,19 @@ export const requiredFile = val => {
   return false
 }
 
-export const validDate = val => {
+export const validDate = (val, enableFutureCheck) => {
   const dateObj = parse_date(val)
   if (!val || !isValid(dateObj)) {
     return false
   }
   if (format(dateObj, 'YYYY-MM-DD') !== val) {
     return false
   }
-  if (isFuture(val)) {
+  if (enableFutureCheck === undefined || enableFutureCheck) {
+    if (isFuture(val)) {
+      return true
+    }
```

Comment:
This might need to return false.
```javascript
return isFuture(val) ? true : false

Reply:
it will be redundant to the default return at the end of the function